src/app/app-routing.module.ts

```diff
@@ -1,7 +1,9 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: 'profile', loadChildren: './player-profile/player-profile.module#PlayerProfileModule' }
+];
```

Comment:
you don't need app-routing.module.ts. It should be app-routing.config.ts

Here you should keep only routes definition. 

` imports: [RouterModule.forRoot(routes)] ` should be defined in app.module.ts